static/app/views/organizationIntegrations/integrationItem.tsx

```diff
@@ -1,46 +1,38 @@
-import {Component} from 'react';
 import styled from '@emotion/styled';
 
 import space from 'sentry/styles/space';
 import {Integration} from 'sentry/types';
 import IntegrationIcon from 'sentry/views/organizationIntegrations/integrationIcon';
 
-type DefaultProps = {
-  compact: boolean;
-};
-
-type Props = DefaultProps & {
+type Props = {
   integration: Integration;
+  compact?: boolean;
 };
-export default class IntegrationItem extends Component<Props> {
-  static defaultProps: DefaultProps = {
-    compact: false,
-  };
 
-  render() {
-    const {integration, compact} = this.props;
-    return (
-      <Flex>
-        <div>
-          <IntegrationIcon size={compact ? 22 : 32} integration={integration} />
-        </div>
-        <Labels compact={compact}>
-          <IntegrationName data-test-id="integration-name">
-            {integration.name}
-          </IntegrationName>
-          <DomainName compact={compact}>{integration.domainName}</DomainName>
-        </Labels>
-      </Flex>
-    );
-  }
+function IntegrationItem({integration, compact = false}: Props) {
+  return (
+    <Flex>
+      <div>
+        <IntegrationIcon size={compact ? 22 : 32} integration={integration} />
+      </div>
+      <Labels compact={compact}>
+        <IntegrationName data-test-id="integration-name">
+          {integration.name}
+        </IntegrationName>
+        <DomainName compact={compact}>{integration.domainName}</DomainName>
+      </Labels>
+    </Flex>
+  );
 }
 
+export default IntegrationItem;
```

Comment:
nit: can we make this a "named export" ?

Reply:
we can I just didn't want to do it here in the conversion